src/main.js

```diff
@@ -69,15 +70,19 @@ let permissionManager = null;
 const argv = parseArgv(process.argv.slice(1));
 const hideOnStartup = shouldBeHiddenOnStartup(argv);
 
-if (argv['data-dir']) {
+if (argv['portable']) {
```

Comment:
ESLint error: ["portable"] is better written in dot notation (dot-notation)

`argv['portable']` -> `argv.portable`